scripts/ziti-builder.sh

```diff
@@ -125,10 +126,10 @@ function main() {
             -E make_directory \
             ./build  
         cmake \
-            --preset "${CMAKE_PRESET:-ci-linux-x64}" \
+            --preset "${CMAKE_PRESET:-ci-linux-x64-static-libssl}" \
```

Comment:
Do we want this? I would think we want to let the build use the system's shared libssl.so if someone is building their own binary.